demisto_sdk/commands/init/initiator.py

```diff
@@ -378,6 +380,31 @@ def get_valid_user_input(options_list: List[str], option_message: str) -> str:
             except ValueError:
                 user_input = input("\nThe option must be number, please enter valid choice: ")
 
+    def ignore_secrets_automatically(self):
+        pack_dir = get_pack_name(self.full_output_path)
+        files_and_directories = glob.glob(f'{self.full_output_path}/**/*', recursive=True)
+
+        sv = SecretsValidator(white_list_path='./Tests/secrets_white_list.json')
+        # remove directories and irrelevant files
+        files = [f for f in files_and_directories if os.path.isfile(f) and sv.is_text_file(f)]
+        all_secrets = sv.search_potential_secrets(files)  # -> {'a': {'b': ['c', 'd'], 'e': ['f']}, 'g': ['h']}
+        secrets: set = set()
+
+        def extract_values_from_nested_dict_to_a_set(d, s):
```

Comment:
Please move this method outside the function, provide a docstring with examples and provide better arguments and variables names.
v/d/i/s are not informative and are hard to follow.